.github/workflows/provisioning-tests.yaml

```diff
@@ -75,4 +75,37 @@ jobs:
           V2PROV_TEST_RUN_REGEX: "^Test_Operation_SetA_.*$"
           KDM_TEST_K8S_MINOR: ${{ matrix.k8s-minor }}
           PREV_COMMIT_PR_SHA: ${{ github.event.pull_request.base.sha }}
+          PREV_COMMIT_PUSH_SHA: ${{ github.event.before }}
+
+  provisioning-operations-test-b:
```

Comment:
Wondering if we can avoid copy-paste here by introducing another dimension for the `matrix` to specify `V2PROV_TEST_RUN_REGEX` ? (If this works we could also roll all tests into `provisioning-test` job by covering `"^Test_Provisioning_.*$"` regex as well).
If we proceed with that consolidation, please let me know as I would need to update branch protection rules to no longer require `provisioning-operations-test` to succeed to merge PRs.